anchorecli/cli/system.py

```diff
@@ -486,3 +486,51 @@ def delete_data(feed, group=None):
             ecode = 2
 
     anchorecli.cli.utils.doexit(ecode)
+
+
+@system.group(name="webhook", short_help="For testing webhooks")
+def webhook():
+    pass
+
+
+@webhook.command(name="test", short_help="Test that a given webhook works")
+@click.argument("webhook_type", nargs=1, required=False)
+@click.option(
+    "--ntype",
+    default="tag_update",
+    help="The type of notification to send via the given webhook_type",
+    required=False,
+)
+def test_webhook(webhook_type, ntype):
+    """
+    Call the Test Webhook Endpoint on Anchore Engine
+    :param webhook_type: the type of webhook to send a test notification for
+    """
+
+    ecode = 0
+
+    if not webhook_type:
+        webhook_type = "general"
```

Comment:
nit: this can be set as a default in the click arg definition I believe, then it's also clear to users in the help output.